frontend/packages/shipwright-plugin/console-extensions.json

```diff
@@ -117,6 +220,21 @@
       },
       "component": { "$codeRef": "pages.BuildDetailsPage" }
     },
+    "flags": {
+      "required": ["SHIPWRIGHT_BUILD_V1ALPHA1"],
+      "disallowed": ["SHIPWRIGHT_BUILDRUN"]
+    }
```

Comment:
Same here. Can we test it everything works as expected without this feature flags?